metriq-api/api-routes.js

```diff
@@ -39,8 +39,16 @@ router.route('/recover')
   .post(accountController.recover)
 router.route('/password')
   .post(accountController.password)
+const rateLimit = require('express-rate-limit');
```

Comment:
This should be moved to the top of the file along with other requires

Reply:
Good point. Done in ae7473674b1dcbabb45af580afebc8fd533a622d.